src/components/Main/TaskList/Task/Task.tsx

```diff
@@ -1,36 +1,44 @@
-import React from 'react'
-import classes from './Task.module.scss'
-import ThreeDotsTask from '../../../../assets/img/vertical-ellipsis.png';
+import React from "react";
+import classes from "./Task.module.scss";
+import ThreeDotsTask from "../../../../assets/img/vertical-ellipsis.png";
 
 type MyProps = {
-  taskIndex: number
-  name: string
-  handleActivateTask: (params: number) => void
-  activeTask: number | null
-}
+  taskIndex: number;
+  name: string;
+  handleActivateTask: (params: number) => void;
+  activeTask: number | null;
+};
 
-type MyState = {
-}
+type MyState = {};
 
-export default class Task extends React.PureComponent<MyProps, MyState> {
-  constructor(props: MyProps){
-    super(props)
-  }
-
-  render() {
-    return (
-      <div className={`${classes.task} ${this.props.activeTask === this.props.taskIndex ? classes['is-active'] : '' }`} onClick={() => this.props.handleActivateTask(this.props.taskIndex)}>
-        <div className={classes['task-information']}>
-          <div className={classes['complete-task']}></div>
-          <span>{this.props.name}</span>
-        </div>
-        <div className={classes['actions']}>
-          <span>0/1</span>
-          <div className={classes['task-button']}>
-            <img className={classes['button-img']}src={ThreeDotsTask} alt="three-dots"/>
-          </div>
+const Task: React.FC<MyProps> = ({
+  taskIndex,
+  name,
+  handleActivateTask,
+  activeTask,
+}) => {
+  return (
+    <div
+      className={`${classes.task} ${
+        activeTask === taskIndex ? classes["is-active"] : ""
+      }`}
```

Comment:
te recomiendo usar el package llamado [classnames](https://www.npmjs.com/package/classnames) así las clases condicionales las puedes manejar de esta forma:

```suggestion
      className={classNames(classes.task, { classes["is-active"]: taskIndex })}
```